README.md

```diff
@@ -10,7 +10,7 @@ Full documentation is available on the [JHipster documentation for microservices
 
 Click this button to deploy your own instance of the registry:
 
-[![Deploy to Heroku](https://camo.githubusercontent.com/c0824806f5221ebb7d25e559568582dd39dd1170/68747470733a2f2f7777772e6865726f6b7563646e2e636f6d2f6465706c6f792f627574746f6e2e706e67)](https://dashboard.heroku.com/new?&template=https%3A%2F%2Fgithub.com%2Fjhipster%2Fjhipster-registry)
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
```

Comment:
Thanks, this is great... And now I'm asking you exactly because I'm writing the docs, where it won't work, can you give a correct URL? Don't do a PR, I'm refactoring everything at the moment :-)